Requirements/ReqIFDal/ReqIFBuilder.cs

```diff
@@ -300,15 +300,10 @@ private void InstantiateRequirementType()
             var requirements = this.exportedIteration.RequirementsSpecification.SelectMany(s => s.Requirement);
             foreach (var requirement in requirements)
             {
+                // TODO: short term fix. The intent of reuse of spec type with use of applied rules is a bit unintuitive and convoluted without clear reasoning. Needs to be completely looked over.
```

Comment:
please add a GH issue for this and put the url of it in the code so we do not forget. a TODO will be forgotten.